content/graphql/resolvers-map.md

```diff
@@ -308,19 +308,23 @@ type Query {
 
 #### Class inheritance
 
-Args:
+You can use standard TypeScript class inheritance to create base classes with generic utility type features (fields and field properties, validations, etc.) that can be extended. For example, you may have a set of pagination related arguments that always include the standard `offset` and `limit` fields, but also other index fields that are type-specific. You can set up a class hierarchy as shown below.
+
+Base `@ArgsType()` class:
 
 ```typescript
 @ArgsType()
 class PaginationArgs {
-  @Field(type => Int)
+  @Field((type) => Int)
```

Comment:
For the sake of consistency, should we remove parenthesis? `(type)` -> `type` (see other examples above)

Reply:
Yes.  Do you prefer to keep it as (type => Int) vs. () => Int?  I prefer the second myself, as I was initially confused about whether some argument is passed, but I'm OK either way.

Reply:
TypeORM convention is `type => Int`. Perhaps, we should stick to this approach(?) (to be coherent)